src/test/resources/org/jabref/logic/importer/fileformat/MsBibImporterTest7.bib

```diff
@@ -5,7 +5,7 @@ @Article{Orlowski;2011
   title        = {Application of Ontology In the ITIL Domain},
   journal      = {Information Systems Architecture and Technology: Service Oriented Networked Systems},
   year         = {2011},
-  month 	   = {mar},
+  month 	   = {03},
```

Comment:
It's only, because many styles define the string `mar`. At least this is what I read from the discussion at https://tex.stackexchange.com/q/70455/9075.